zerver/openapi/zulip.yaml

```diff
@@ -9901,25 +9901,33 @@ paths:
                       subscribed:
                         type: object
                         description: |
-                          A dictionary where the key is the Zulip API email address of the user/bot
+                          A dictionary where the key is the id of the user/bot
                           and the value is a list of the names of the channels that were subscribed
                           to as a result of the query.
+
+                          **Changes**: Before Zulip 10.0 (feature level 288), the
+                          key of the dictionary is the Zulip API email address of the user/bot.
+
                         additionalProperties:
                           description: |
-                            `{email_address}`: List of the names of the channels that were subscribed
+                            `{id}`: List of the names of the channels that were subscribed
                             to as a result of the query.
```

Comment:
Hmm, reading this, should we change the channels to be referred to by ID too? I guess maybe we can't readily do that, until we change the client to pass channel IDs, which would require decoupling channel creation from channel subscribe operations. So this is fine.